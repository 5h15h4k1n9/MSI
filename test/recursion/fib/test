fibonacci(0)
fibonacci(5)
fibonacci(15)
exit
