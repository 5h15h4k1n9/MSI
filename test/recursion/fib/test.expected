(int) 0
(int) 5
(int) 610
Bye-bye!
